lib/core/src/clientLogin.cpp

```diff
@@ -290,14 +289,14 @@ int clientLogin(rcComm_t* _comm, const char* _context, const char* _scheme_overr
         return SYS_INVALID_INPUT_PARAM;
     }
 
-    static constexpr auto minimum_version_for_auth_plugin_framework = irods::version{4, 3, 0};
-    const auto server_version = irods::to_version(_comm->svrVersion->relVersion);
-    if (!server_version) {
-        static constexpr auto ec = VERSION_EMPTY_IN_STRUCT_ERR;
-
-        const std::string msg = fmt::format(
-            "Failed to get version from server [{}]\n",
-            _comm->svrVersion->relVersion);
+    try {
+        if (!irods::experimental::auth::use_legacy_authentication(*_comm)) {
```

Comment:
I'm not even sure whether this is necessary given the direction we are moving with respect to newer clients and older servers (that is, they will not be made compatible).

Reply:
We should leave support for the older authentication flow until we determine it is no longer needed, at which point we should probably just stop using `clientLogin`.